src/lib/dataEntities/networks.ts

```diff
@@ -78,7 +80,7 @@ export interface EthBridge {
 }
 
 export interface L1Networks {
-  [id: string]: L1Network
+  [id: string]: L1Network | L2Network
```

Comment:
Wouldn't this change be considered too breaking?

Reply:
Shouldn't be, as it's a union type so a superset of the previous one

Reply:
What if you try to access properties that are on L1Network but not on L2Network? You'll have to check for undefined first as typescript will complain.

Reply:
Let's create a type `ParentChains` that could either be `L1Network` or `L2Network`, and `Chains` that can be `L2Network`.

We should also create separate `getParentChain` and `getChain` instead of editing `getL1Network` and `getL2Network`

Reply:
We'd also need `parentChains` that mimics `l1Networks` but adds L2s that are parent chains. We also need to use that when instantiating a bridger class